examples/backup_vm.py

```diff
@@ -221,7 +221,7 @@ def cmd_download(args):
 
         backup = get_backup(connection, backup_service, args.backup_uuid)
         if backup.phase != types.BackupPhase.READY:
-            raise RuntimeError("Backup {} is not ready".format(backup_uuid))
+            raise RuntimeError("Backup {} is not ready (current phase is {}).".format(args.backup_uuid, backup.phase))
```

Comment:
There is an attempt to avoid long lines (>=80).
Thought there are a few lines in the file that are longer, but can you split please :)